app/models/embed/purl.rb

```diff
@@ -266,8 +287,10 @@ def image?
           mimetype =~ %r{image/jp2}i
         end
 
+        # Return the size of the file in bytes as an integer or nil if it is not present.
```

Comment:
How about we never return `nil`? That means we don't have to do any guarding?

Reply:
I think there are cases where there actually is no value, which some of the logic depends on. I initially had it return 0 when it was undefined and some tests broke.